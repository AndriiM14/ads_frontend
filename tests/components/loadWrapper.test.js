import React from "react";
import { render, screen  } from "@testing-library/react";
import LoaderWrapper from "../../src/components/loaderWrapper";

describe('LoaderWrapper component tests', () => {
    test("Test", () => {
        const { asFragment } = render(<LoaderWrapper />);
        expect(asFragment).toMatchSnapshot();
    });
});