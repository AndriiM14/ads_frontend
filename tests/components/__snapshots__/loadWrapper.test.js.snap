// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderWrapper component tests Test 1`] = `[Function]`;
